### ISSUE TITLE
pretrained model

### ISSUE THREAD

**Issue by catsglasses** [hzh8311/pyhowfar#1]
Hi, 

Very great job, thanks for providing the pytorch version. How about the accuracy compared with the orginal torch version? Can you share a pretrained model for me?

Thanks a lot

**Comment by hzh8311** [comment]
There is a gap between this implement with the offical Torch version, and I have no plan to maintain this repo recently.
The pretrained model is lost when I graduated  from my college.
